test/e2eTests/pages/registerPatientPage.js

```diff
@@ -45,13 +45,18 @@ class RegisterPatientPage extends Page {
       neighborhood: { css: '#address5' },
       cell: { css: '#address3' },
       street: { css: '#address1' },
-      provenience: { css: 'form select[id=Proveniência]' },
+      provenience: 'select[id="d10628a7-ba75-4495-840b-bf6f1c44fd2d"]',
       reference: { css: '#Ponto de Referência' },
       phone1: { css: '#Numero de Telefone 1' },
       phone2: { css: '#Numero de Telefone 2' },
-      hivTestDate: { css: '#Data do teste HIV' },
-      testType: { css: '#Tipo de teste HIV' },
-      testResult: { css: '#Resultado do Teste HIV' }
+      hivTestDate: 'input[id="46e79fce-ba89-4ec9-8f31-2dfd9318d415"',
```

Comment:
Will these values always be the same?

Reply:
Seems like it would be safer to use css here. Can you add an attribute to the html, maybe something like `<div ... qa-hivTestDate />` or something like that?

Reply:
Yes. I have already edited some of the html to replace name uuid.